conjur_api/http/api.py

```diff
@@ -156,6 +156,34 @@ async def resources_list(self, list_constraints: dict = None) -> dict:
         # ?tocpath=Developer%7CREST%C2%A0APIs%7C_____17
         return resources
 
+    async def check_privilege(self, kind: str, resource_id: str, privilege: str, role_id: str) -> bool:
+        """
+        This method is used to check for a privilege on a resource.
+        """
+        params = {
+            'account': self._account,
+            'kind': kind,
+            'identifier': resource_id,
+            'privilege': privilege,
+            'role': role_id if role_id else "",
+        }
+        params.update(self._default_params)
+
+        try:
+            await invoke_endpoint(HttpVerb.HEAD, ConjurEndpoint.PRIVILEGE,
```

Comment:
I see you're using an HTTP HEAD request, which is intriguing because while the [docs](https://docs.conjur.org/Latest/en/Content/Developer/Conjur_API_Check_Permission.htm?tocpath=Developer%7CREST%C2%A0APIs%7C_____21#URI) do say to use GET, there is generally no response body (hence the 204 status code). So while I think HEAD would work, maybe we ought to use GET to stay consistent with the docs.